packages/api-server/src/modules/workspace/index.ts

```diff
@@ -29,33 +33,47 @@ export class WorkspaceController {
     const config = DConfig.readConfigSync(uri);
     let engine;
     if (config.dev?.enableEngineV3) {
-      engine = DendronEngineV3.create({
-        wsRoot: uri,
-        logger,
-      });
+      if (config.dev?.useSqlite) {
+        engine = await DendronEngineV3Factory.createWithSqliteStore({
+          wsRoot: uri,
+          logger,
+        });
+      } else {
+        engine = DendronEngineV3.create({
+          wsRoot: uri,
+          logger,
+        });
+      }
     } else {
       engine = DendronEngineV2.create({
         wsRoot: uri,
         logger,
       });
     }
-    const { data, error } = await engine.init();
-    if (error && error.severity === ERROR_SEVERITY.FATAL) {
-      logger.error({ ctx, msg: "fatal error initializing notes", error });
-      return { data, error };
-    }
-    await putWS({ ws: uri, engine });
-    const duration = getDurationMilliseconds(start);
-    logger.info({ ctx, msg: "finish init", duration, uri, error });
-    let error2;
-    if (error) {
-      error2 = error2PlainObject(error);
+
+    // SQLite doesn't need to do engine.init()
```

Comment:
Just heads up that there are 2 other places were engine.init is called.

1) Dendron-cli command. Right now we have flag for using v2 or v3. Not sure if we wanted to update that to use sqllite as well.

2) After running `engine.deleteSchema`, we call init again. This might cause a temp problem since we'll end up tearing down the db every time but I'm guessing that won't be necessary once the sqllite schema logic is fleshed out

Reply:
Added todo comments in both places for now.